pkg/configfilearg/parser.go

```diff
@@ -161,26 +162,28 @@ func (p *Parser) findOverrideFlag(args []string) (string, bool) {
 	return "", false
 }
 
-func (p *Parser) findConfigFileFlag(args []string) (string, bool) {
+// findConfigFileFlag returns the value of the config file env var or CLI flag.
+// If neither are set, it returns the default value.
+func (p *Parser) findConfigFileFlag(args []string) string {
 	if envVal := os.Getenv(p.EnvName); p.EnvName != "" && envVal != "" {
-		return envVal, true
+		return envVal
 	}
 
 	for i, arg := range args {
 		for _, flagName := range p.ConfigFlags {
 			if flagName == arg {
 				if len(args) > i+1 {
-					return args[i+1], true
+					return args[i+1]
 				}
 				// This is actually invalid, so we rely on the CLI parser after the fact flagging it as bad
-				return "", false
+				return ""
```

Comment:
I guess this is fine, since we didn't actually do anything if the value was false?

Reply:
yeah, and the callers all handle an empty string properly anyways